src/server/network/protocol/protocolgame.cpp

```diff
@@ -3434,11 +3539,64 @@ void ProtocolGame::sendCyclopediaCharacterItemSummary() {
 	msg.addByte(0xDA);
 	msg.addByte(CYCLOPEDIA_CHARACTERINFO_ITEMSUMMARY);
 	msg.addByte(0x00);
-	msg.add<uint16_t>(0);
-	msg.add<uint16_t>(0);
-	msg.add<uint16_t>(0);
-	msg.add<uint16_t>(0);
-	msg.add<uint16_t>(0);
+
+	// Inventory
+	auto startInventoryItems = msg.getBufferPosition();
+	uint16_t inventoryItems = 0;
+	msg.skipBytes(2);
+
+	for (std::underlying_type<Slots_t>::type slot = CONST_SLOT_FIRST; slot <= CONST_SLOT_AMMO; slot++) {
+		Item* inventoryItem = player->getInventoryItem(static_cast<Slots_t>(slot));
+		if (inventoryItem) {
+			++inventoryItems;
+			msg.add<uint16_t>(inventoryItem->getID());
+			msg.add<uint32_t>(inventoryItem->getItemCount());
+		}
+		// TODO show / count items in backpack
+	}
+
+	// Stash
+	// TODO Why does it display items in the store inbox?
```

Comment:
yeah i noticed this too. I'll help debug later but damn this PR has a lot of stuff